frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -12,7 +13,17 @@ const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state, dataTest }) => (
 );
 
 const HealthItem: React.FC<HealthItemProps> = React.memo(
-  ({ className, state, title, details, popupTitle, noIcon = false, icon, children }) => {
+  ({
+    className,
+    state,
+    title,
+    details,
+    popupTitle,
+    noIcon = false,
+    icon,
+    children,
+    isHCPresent,
```

Comment:
```suggestion
    position,
```
Better to pass a `prop` component for custom `position`.